src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_INullCheckedParameters.cs

```diff
@@ -726,7 +726,7 @@ class C
 {
     public void M()
     {
-        Func<string, string> func1 = _! => null;
+        Func<string, string> func1 = _!! => null;
```

Comment:
This reminds me that we need to figure out what this does with multiple discard parameters in lambdas. I added a note to the test plan: [https://github.com/dotnet/roslyn/issues/36024#issuecomment-670174165](https://github.com/dotnet/roslyn/issues/36024#issuecomment-670174165)